tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1665,4 +1665,70 @@ public function theAdministratorTriesToAddTheFollowingUsersToAGroupAtOnceUsingTh
 		}
 		$this->addMultipleUsersToGroup($user, $userIds, $groupId, $table);
 	}
+
+	/**
+	 * @When user :user gets details of the group :groupName using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $groupName
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfTheGroupUsingTheGraphApi(string $user, string $groupName): void {
+		$groupId = $this->getGroupIdByName($groupName);
+		$this->getDetailsOfGroup($user, $groupId);
+	}
+
+	/**
+	 * @When user :user gets details of non-existing group using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsDetailsOfNonExistingGroupUsingTheGraphApi(string $user): void {
+		// generates random group id for non-existing group
+		$groupId = WebDavHelper::generateUUIDv4();
+		$this->getDetailsOfGroup($user, $groupId);
+	}
+
+	/***
+	 * @param string $user
+	 * @param string $groupId
+	 *
+	 * @return void
+	 *
+	 * @throws GuzzleException
+	 */
+	public function getDetailsOfGroup(string $user, string $groupId): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+
+		$this->featureContext->setResponse(
+			GraphHelper::getGroup(
+				$this->featureContext->getBaseUrl(),
+				$groupId,
+				$this->featureContext->getStepLineRef(),
+				$credentials["username"],
+				$credentials["password"]
+			)
+		);
+	}
+
+	/**
+	 * Get Group Id from response
+	 *
+	 * @param string $groupName - name of the group
+	 *
+	 * @return string - id of the group
+	 * @throws Exception | GuzzleException
+	 */
+	public function getGroupIdByName(string $groupName): string {
+		$allGroups = $this->getArrayOfGroupsResponded($this->listGroups());
+		foreach ($allGroups as $group) {
+			if ($group['displayName'] === $groupName) {
+				return $group['id'];
+			}
+		}
+		throw new Exception("Group with name '$groupName' not found in the groups list");
+	}
```

Comment:
i think we can replace this function with `$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");` could you check it?

Reply:
We already have a function that does the same thing here https://github.com/owncloud/ocis/blob/fe4195d69410c0a13164d96d8d0770bec893995f/tests/acceptance/features/bootstrap/FeatureContext.php#L4603

I think `getAttributeOfCreatedGroup` is more about getting the attributes of group created in the previous steps and the above function gets the id directly from a request-response so it might be more appropriate. Still, please check which one fits in your case.